### PR TITLE
Various fixes to the ADC-branch

### PR DESCRIPTION
**Pull request by SkyToGround**
### Issue reference / description

The branch you merge from should already reference an event-formation-unit github ticket number. You can add a descriptive title, but if an issue is referenced, you don't have to.

## Checklist for submitter

- [ ] Check for conflict with integration test
- [ ] Unit tests pass

---

## Nominate for Group Code Review (Anyone can nominate it)
Indicate if you think the code should be reviewed in a Thursday code review session.

- [ ] Recommend for group code review

Also, nominate it on the code_review Slack channel.

### CODE REVIEW

File: src/adc_readout/PeakFinder.cpp [hunk 1]
```diff
@@ -20,17 +20,20 @@ void PeakFinder::processData(SamplingRun const &Data) {
...
             std::to_string(Data.Identifier.ChannelNr),
-        100, 50ms, ProducerPtr.get(),
-        EventSerializer::TimestampMode::INDEPENDENT_EVENTS);
+        200, 500ms, ProducerPtr.get(),
+        EventSerializer::TimestampMode::TIME_REFERENCED);
   }
   auto &CurrentSerialiser = Serialisers[Data.Identifier];
   CurrentSerialiser->addReferenceTimestamp(
       Data.ReferenceTimestamp.getTimeStampNS());
+  auto EventId =
```
Comment (mortenjc) [inline src/adc_readout/PeakFinder.cpp]: I'n not sure what happens here. Seems like you're calculating a pixelid outside the 
defined geometry range? If the 512x512 here are related to essgeometry
then you should use essgeometry.nx() and essgeometry.ny() instead.

Comment (SkyToGround) [reply src/adc_readout/PeakFinder.cpp]: This particular detector does not have any geometry. It is a one pixel monitor. The event-id you see here is based on a request (demand?) from @matthew-d-jones.

File: src/adc_readout/PeakFinder.cpp [hunk 2]
```diff
@@ -20,17 +20,20 @@ void PeakFinder::processData(SamplingRun const &Data) {
...
+      512u * 512u + static_cast<std::uint32_t>(Data.Identifier.ChannelNr);
```
Comment (mortenjc) [inline src/adc_readout/PeakFinder.cpp]: variable name just for clarity?

Comment (SkyToGround) [inline src/adc_readout/PeakFinder.cpp]:
```suggestion
      PixelIdOffset + static_cast<std::uint32_t>(Data.Identifier.ChannelNr);
```

File: src/adc_readout/PeakFinder.cpp (hunk 1 above)
Comment (SkyToGround) [inline src/adc_readout/PeakFinder.cpp]:
```suggestion
  constexpr std::uint32_t PixelIdOffset{512*512};
  auto EventId =
```

**Pull request merged**